tasks/src/test/java/yandex/WebDriverYandexDiskUserAccessTest.java

```diff
@@ -19,9 +23,15 @@
         {UserFactory.withValidCredentialsFromProperty(), true}};
   }
 
+  @BeforeClass
+  public void prepareTest() {
```

Comment:
I'd move the method to a separate base test and reuse it in child classes. Or probably a better idea is to manage driver initialization and closing in a separate service class, e.g. DriverManager. The class is supposed to be used in Page Objects only, not on a test layer